index.html

```diff
@@ -374,7 +374,7 @@
 					<dt>primary pen input</dt><dd>A pointer representing pen input is considered the <i>primary pen input</i> if its <code>pointerdown</code> event was dispatched when no other active pointers representing pen input existed.</dd>
 				</dl>
 				<div class="note" id="multiple-primary-pointers">When two or more pointer device types are being used concurrently, multiple pointers (one for each <code>pointerType</code>) are considered primary. For example, a touch contact and a mouse cursor moved simultaneously will produce pointers that are both considered primary.</div>
-				<div class="note">In the case where there are multiple <a href="#the-primary-pointer">primary pointers</a>, 
+				<div class="note">In the case where there are multiple <a data-lt="The Primary Pointer">primary pointers</a>, 
```

Comment:
Oh, it looks like the uses of `<a>primary pointer</a>` I was looking at are actually broken!  I think that's just because `<dfn>The Primary Pointer</dfn>` should be `The <dfn>Primary Pointer</dfn>`.  If you change that, then I think you can avoid the `data-lt` here and elsewhere.